apps/fluent-tester/src/TestComponents/Text/consts.ts

```diff
@@ -2,9 +2,17 @@ export const HOMEPAGE_TEXT_BUTTON = 'Homepage_Text_Button';
 export const TEXT_TESTPAGE = 'Text_TestPage';
 
 /* E2E Testing Text 1 */
-export const FIRST_TEXT_COMPONENT = 'Text_Test_Component'; // A component on each specific test page
+export const FIRST_TEXT_COMPONENT = 'Text_Test_Component';
 export const TEXT_ACCESSIBILITY_LABEL = 'E2E testing Text_Test_Component accessibility label';
 
 /* E2E Testing Text 2 */
-export const SECOND_TEXT_COMPONENT = 'Text_No_A11y_label_Component';
-export const TEXT_COMPONENT_CONTENT = 'Test Text2 - No Accessibility Label'; // A component on each specific test page
+export const SECOND_TEXT_COMPONENT = 'Text_No_A11y_Label_Component';
```

Comment:
Why does deprecated Text have more constants than V1 text?

Reply:
The @fluentui-react-native/text package exports two versions of Text: [TextV1](https://github.com/microsoft/fluentui-react-native/blob/main/packages/components/text/src/index.ts#L1) and the [deprecated Text](https://github.com/microsoft/fluentui-react-native/blob/main/packages/components/text/src/index.ts#L26) component.